common/app/common/NewNavigation.scala

```diff
@@ -462,19 +462,29 @@ object NewNavigation {
       SubSectionLink("world", worldSubNav),
       SubSectionLink("money", moneySubNav),
       SubSectionLink("football", footballSubNav),
-      SubSectionLink("todayspaper", todaysPaperSubNav),
-      SubSectionLink("theobserver", theObserverSubNav),
+      SubSectionLink("theguardian", todaysPaperSubNav),
+      SubSectionLink("observer", theObserverSubNav),
       SubSectionLink("crosswords", crosswordsSubNav)
     )
 
     def isEditionalistedSubSection(sectionId: String) = {
       editionalisedSubSectionLinks.exists(_.pageId == sectionId)
     }
 
+    val frontLikePages = List(
+      "theguardian",
+      "observer",
+      "football/live",
+      "football/tables",
+      "football/competitions",
+      "football/results",
+      "football/fixtures"
+    )
+
     def getSectionOrTagId(page: Page) = {
       val tags = Navigation.getTagsFromPage(page)
       val commonKeywords = tagPages.intersect(tags.keywordIds)
-      val isTagPage = page.metadata.isFront && tagPages.contains(page.metadata.id)
+      val isTagPage = (page.metadata.isFront || frontLikePages.contains(page.metadata.id)) && tagPages.contains(page.metadata.id)
```

Comment:
Nope. Two different checks for two different things.

This is just determining whether we need to use the section id, the page id, or something else to determine how we identify the page.

The second use is determining whether we give it a subnav list for fronts, or for articles. They differ because we don't show tertiary on articles